src/Form/Control/AddressLookup.php

```diff
@@ -106,7 +105,7 @@ public function renderView(): void
     {
         JsLoader::load($this->getApp());
 
-        $this->js(true)->atkAddressLookup($this->getLookupSettings());
+        $this->js(true)->AddressLookup($this->getLookupSettings());
```

Comment:
where does the `AddressLookup` JS comes from? In atk4/ui AFAK there is no such plugin